Missing dependency lodash-es

### Describe the bug

Module not found: Error: Can't resolve `lodash-es` in `.../node_modules/@microsoft/fast-web-utilities/dist`

### To reproduce

1. Install the dependency to a VS Code extension with webpack bundling;
2. Start building your project
3. See it fail

### Expected behavior

The expected behavior would be that the `lodash-es` is defined as a dependency of the library, this way, it will not fail.

Also, the `FAST` docs recommend it: https://www.fast.design/docs/integrations/react#configuring-packages

### Screenshots

```bash
ERROR in ./node_modules/@microsoft/fast-web-utilities/dist/class-names.js
    Module not found: Error: Can't resolve 'lodash-es' in '.../node_modules/@microsoft/fast-web-utilities/dist'
     @ ./node_modules/@microsoft/fast-web-utilities/dist/class-names.js 1:0-49 7:14-24 9:18-26
     @ ./node_modules/@microsoft/fast-web-utilities/dist/index.js
     @ ./node_modules/@microsoft/fast-foundation/dist/esm/utilities/direction.js
     @ ./node_modules/@microsoft/fast-foundation/dist/esm/utilities/index.js
     @ ./node_modules/@microsoft/fast-foundation/dist/esm/index.js
     @ ./node_modules/@vscode/webview-ui-toolkit/dist/esm/checkbox/index.js
     @ ./src/panelWebView/index.tsx
```

### Desktop (please complete the following information):

-   OS Version: macOS
-   Toolkit Version: 0.8.1

--- a/package.json
+++ b/package.json
@@ -31,7 +31,8 @@
 	},
 	"dependencies": {
 		"@microsoft/fast-element": "^1.2.0",
-		"@microsoft/fast-foundation": "^1.24.7"
+		"@microsoft/fast-foundation": "^1.24.7",
+		"lodash-es": "^4.17.21"
 	},
 	"devDependencies": {
 		"@babel/core": "^7.14.3",
@@ -57,7 +58,6 @@
 		"eslint-config-prettier": "^8.1.0",
 		"gh-pages": "^3.1.0",
 		"jest": "^27.2.4",
-		"lodash-es": "^4.17.21",
 		"prettier": "^2.2.1",
 		"rollup": "^2.40.0",
 		"rollup-plugin-delete": "^2.0.0",

